lib/strings.nix

```
@@ -206,6 +206,18 @@ rec {
   */
   makeLibraryPath = makeSearchPathOutput "lib" "lib";
 
+  /* Construct a include search path (such as C_INCLUDE_PATH) containing the
+     includes for a set of packages
```

```suggestion
     header files for a set of packages or paths.
```

clarify wording slightly